opendevin/sandbox/e2b/README.md

```diff
@@ -0,0 +1,35 @@
+# How to use E2B
+
+[E2B](https://e2b.dev) is an [open-source](https://github.com/e2b-dev/e2b) secure cloud environment (sandbox) made for running AI-generated code and agents. E2B offers [Python](https://pypi.org/project/e2b/) and [JS/TS](https://www.npmjs.com/package/e2b) SDK to spawn and control these sandboxes.
+
+## Getting started
+
+1. [Get your API key](https://e2b.dev/docs/getting-started/api-key)
+
+1. Set your E2B API key in the `config.toml` file as `E2B_API_KEY`
```

Comment:
We're mostly getting rid of `config.toml` now that everything is running inside docker. Maybe this should be passed into `docker run` with `-e`?

Reply:
Yeah, that should work, I can update it. So `config.py` isn't used anymore and instead we should pass the env vars, right?

Reply:
I updated the readme to instruct users to set the E2B API key to the `E2B_API_KEY` env var. 

If there's nothing else you think should change, this PR should be good to go!